test_util/helpers.py

```diff
@@ -76,7 +80,13 @@ def api_request(self, method, path, host_url=None, **kwargs):
         if self.api_base:
             path = path_join(self.api_base, path)
 
-        request_url = path_join(host_url if host_url else self.default_host_url, path)
+        if node is not None:
+            assert host_url is None, 'Cannot set both node and host_url'
```

Comment:
nit: Might make sense to include the values of node and host_url in the assert message